sunkit_pyvista/plotter.py

```diff
@@ -279,7 +288,61 @@ def plot_field_lines(self, field_lines, **kwargs):
             grid = self._coords_to_xyz(field_line.coords.ravel())
             field_line_mesh = pv.StructuredGrid(grid[:, 0], grid[:, 1], grid[:, 2])
             color = {0: 'black', -1: 'tab:blue', 1: 'tab:red'}.get(field_line.polarity)
+            field_line_mesh.add_field_array([color], 'color')
             self.plotter.add_mesh(field_line_mesh, color=color, **kwargs)
             field_line_meshes.append(field_line_mesh)
 
         self._add_mesh_to_dict(block_name='field_lines', mesh=field_line_meshes)
+
+    def save(self, filepath):
+        """
+        Adds all of the meshes in the current dictionary
+        to a :class:`~pyvista.core.MultiBlock`.
+
+        Parameters
+        ----------
+        filepath : `str` or `pathlib.Path`
+            Name of the file to save as, should have vtm or vtmb as an extension.
+
+        Notes
+        -----
+        This saves the rendered plot as a vtm extended file as well as a directory
+        of the individual meshes with the specified name.
+        """
+        file_path = Path(filepath)
+        directory_path = file_path.with_suffix('')
+        if directory_path.exists():
```

Comment:
Do we need to check if the file (as well as the directory) exists too?

Reply:
Yeah, we could specify an `overwrite` keyword to allow for overwriting an existing file and  `allow_save` for saving in an already existing directory, does that sound good?

Reply:
I think just an `overwrite` keyword would be good. Either way, you need to check if the file is present as well right?

Reply:
@nabobalis mentioned that the user might want to save the individual meshes in an existing directory of the same name as that of the vtm file, so I thought it would make sense to have allow_save as well. That way we'd skip the check

Reply:
Or should we raise a warning saying that the directory exists and the files are being stored in there when `overwrite`=True?

Reply:
I dont see the need to raise a warning if they pass in overwrite to be true.

Reply:
But it's not really being overwritten, we write to the existing directory

Reply:
I think we:
- Should error if either the file or directory is currently present
- Not support saving to an existing directory

Reply:
Sure, I shall make this change